src/main/java/com/example/demo/Customer/Customer.java

```diff
@@ -0,0 +1,75 @@
+package com.example.demo.Customer;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
+@Table
+public class Customer {
+    private Integer id;
+    private  String first_name;
+    private  String last_name;
+    private  String email;
+    private  String address;
+    private  Number phone_number;
```

Comment:
Where are the lists of orders, shipments itp? And CardData

Reply:
done (shipment and customer are in one to one relation)